chango/rook-ceph/scripts/rook-ceph-node-cleanup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -x
+
+DEV1=/dev/sda
+DEV2=/dev/sdb
+SSH_CMD='clush -g ruka'
```

Comment:
I'm guessing this script has never been used. I would suggest simply removing it rather than having an incorrect/unsafe script in the repo.

Reply:
Thank you for the feedback, I totally agree. The scripts folder was removed.